cmd/minishift/cmd/util/util.go

```diff
@@ -71,3 +72,17 @@ func UnpackAddons(addonsDir string) error {
 
 	return nil
 }
+
+func ShowProgressDots(progressDots chan bool) {
+	go func() {
+		for {
+			select {
+			case <-progressDots:
+				return
+			default:
+				fmt.Print(".")
+				time.Sleep(2 * time.Second)
```

Comment:
I think we should use 1 second instead of 2 seconds. While testing it I felt the dots were little slow.

Reply:
Tested the updated code and 2 seconds seems fine now.